<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1256</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionDiagram"/>
     <addaction name="actionSession"/>
    </widget>
    <widget class="QMenu" name="menuExport_diagram">
     <property name="title">
      <string>Export diagram</string>
     </property>
     <widget class="QMenu" name="menuLatex">
      <property name="title">
       <string>Latex</string>
      </property>
      <addaction name="actionSource_Latex"/>
      <addaction name="actionSource_to_clipboard"/>
     </widget>
     <addaction name="actionPNG"/>
     <addaction name="menuLatex"/>
    </widget>
    <widget class="QMenu" name="menuExport_all">
     <property name="title">
      <string>Export session</string>
     </property>
     <widget class="QMenu" name="menuLatex_2">
      <property name="title">
       <string>Latex</string>
      </property>
      <addaction name="actionSessionSource"/>
      <addaction name="actionSessionSource_to_clipboard"/>
     </widget>
     <addaction name="actionSessionPNG"/>
     <addaction name="menuLatex_2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="menuExport_diagram"/>
    <addaction name="menuExport_all"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSession_mode"/>
    <addaction name="actionSingle_diagram_mode"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSingle_diagram"/>
    <addaction name="actionAll_session_diagrams"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDiagram">
   <property name="text">
    <string>Diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSession">
   <property name="text">
    <string>Session</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSource_Latex">
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="actionPDF_Latex">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionPNG_Latex">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSource_to_clipboard">
   <property name="text">
    <string>Source to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSessionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionSessionLatexPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionSessionLatexPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionSessionSource">
   <property name="text">
    <string>Source</string>
   </property>
  </action>
  <action name="actionSessionSource_to_clipboard">
   <property name="text">
    <string>Source to clipboard</string>
   </property>
  </action>
  <action name="actionSingle_diagram">
   <property name="text">
    <string>Single diagram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAll_session_diagrams">
   <property name="text">
    <string>All session diagrams</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSession_mode">
   <property name="text">
    <string>Session mode</string>
   </property>
  </action>
  <action name="actionSingle_diagram_mode">
   <property name="text">
    <string>Single diagram mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSource_Latex</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPNG</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportPNG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSession</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newSession()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDiagram</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newDiagram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSource_to_clipboard</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexSourceToClipboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPNG_Latex</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexExportPNG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPDF_Latex</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexExportPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAll_session_diagrams</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSessionMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSingle_diagram</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDiagramMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSessionSource</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSessionSource_to_clipboard</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexSourceToClipboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSessionPNG</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportPNG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSessionLatexPDF</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexExportPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSessionLatexPNG</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>latexExportPNG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>latexSource()</slot>
  <slot>latexSourceToClipboard()</slot>
  <slot>latexExportPNG()</slot>
  <slot>latexExportPDF()</slot>
  <slot>newDiagram()</slot>
  <slot>newSession()</slot>
  <slot>open()</slot>
  <slot>save()</slot>
  <slot>saveAs()</slot>
  <slot>exportPNG()</slot>
  <slot>setDiagramMode()</slot>
  <slot>setSessionMode()</slot>
 </slots>
</ui>
